0-bootstrap/main.tf

```diff
@@ -272,3 +277,16 @@ resource "google_folder_iam_member" "folder_tf_compute_security_resource_admin"
 #   role   = "roles/browser"
 #   member = "serviceAccount:${module.jenkins_bootstrap.jenkins_agent_sa_email}"
 # }
+
+
+#Groups creation resources
+data "google_organization" "org" {
```

Comment:
Can this be a conditional read if var.create_groups_holder.create_groups ?

Reply:
Sure, I'll make sure to change it.